azure-pipelines.yml

```diff
@@ -3,8 +3,8 @@ variables:
   PreviewNumber: $[counter(variables['CurrentSemanticVersionBase'], 1001)]
   CurrentSemanticVersion: '$(CurrentSemanticVersionBase)-preview$(PreviewNumber)'
   NugetPackageVersion: '$(CurrentSemanticVersion)'
-  TOOLKIT_NET_VERSION: '6.0.400'
-  LATEST_NET_VERSION: '6.0.x'
+  TOOLKIT_NET_VERSION: '7.0.x'
```

Comment:
This format '7.0.x' led installing Tizen workload wrong version band. Let me think how we can fix.

Reply:
Just updated the Tizen workload install option which manually defines the sdk version band. I think it's ok to remove the option on MCT.